app/controllers/sequences_controller.rb

```diff
@@ -44,6 +44,38 @@ def show
       @lineages = sequence.lineages(equate_il, true).to_a
     end
 
+    # create list for all ec_cross_reference numbers
+    ecnumbers = []
+    # get all ec_cross_references
+    ec_cross_numbers = @entries.map(&:ec_cross_references)
+    # get all ec_cross_reference numbers and puts them in array
+    ec_cross_numbers.each do |ecs|
+      ecs.each do |ec|
+        ecnumbers << ec.ec_number
+      end
+    end
+    # make list unique
+    uniqecnumbers = ecnumbers.uniq
+    # get all items from table EcNumber
+    ecdb = EcNumber.all
+    # add column names list
+    @column_name = ["name", "digit 1", "digit 2", "digit 3", "digit 4", "count"]
```

Comment:
Use more specific variable name. The lineage table also has column names...


_[Original comment](https://github.ugent.be/unipept/unipept/pull/528#discussion_r3382) by @bmesuere on Wed Oct 21 2015 at 16:45._